config-controller/api/v1alpha1/policy_types.go

```diff
@@ -137,11 +137,27 @@ type MitreAttackVectors struct {
 	Techniques []string `json:"techniques,omitempty"`
 }
 
-// SecurityPolicyStatus defines the observed state of SecurityPolicy
+type SecurityPolicyConditionType string
+
+const (
+	Ready      SecurityPolicyConditionType = "Ready"
+	Reconciled SecurityPolicyConditionType = "Reconciled"
+	Active     SecurityPolicyConditionType = "Active"
+)
+
+// SecurityPolicyCondition defines the observed state of SecurityPolicy
+type SecurityPolicyCondition struct {
+	Type               SecurityPolicyConditionType `json:"type"`
+	Status             bool                        `json:"status"`
```

Comment:
Just quickly looking at other projects' conditions, they seem to use a string instead of bool, but the value of the string is either "True" or "False".